### DISCUSSION TITLE
Season level data fields

### DISCUSSION THREAD

**Discussion by MaunoSeppa** [ZeroQI/Hama.bundle#578] [Q&A] [answered]
As I already gave my thoughts on this [PR](https://github.com/ZeroQI/Hama.bundle/pull/577), I was wondering how to add data fields to the Season level on plex. This is my first time working on the Plex scanners/agents, so I don't really know, where these data fields are defined, but with using ASS and Hama, there are data fields for the title and summary on season level, but they don't get populated even if updated in the Hama.

![vinland season 1 plex](https://github.com/user-attachments/assets/6513bbe9-ccfb-4b75-a68e-e969f3fa359f)

Clip from log to show it populates the summary data for the show:
`metadata.seasons[ 1]-----------------------------------------------------------------------------------------------------------------------------------------
[=] summary                        Sources: MyAnimeList, TheTVDB, AniDB | (MyAnimeList), TheTVDB, AniDB   Inside: '['MyAnimeList', 'AniDB']'  Value: 'Young Thorfinn grew up listening to the stories of old sailors that had traveled..'`

I would like to know if its possible to populate the already existing title and summary and possibly add ranking there. And if possible, where this is done, is it scanner, agent or plex related thing?

**Comment by ZeroQI** [comment, marked as answer]
Plex allow third party agent to use certain fields, but not all fields are able to be updated by agents, from memory season summaries and original title cannot be updated by the agent, the agent will provide it but plex will not use it, but changing it in plex gui will work
